admin/app/adapters/user.js

```diff
@@ -24,6 +24,12 @@ export default class UserAdapter extends ApplicationAdapter {
       const url = this.urlForUpdateRecord(snapshot.id) + '/anonymize';
       return this.ajax(url, 'POST');
     }
+
+    if (snapshot.adapterOptions && snapshot.adapterOptions.dissociate) {
+      const url = this.urlForUpdateRecord(snapshot.id) + '/dissociate';
+      return this.ajax(url, 'PATCH');
```

Comment:
J'aurais plutôt vu un POST mais je crois qu'on ne converge la dessus.

Reply:
J'ai laissé le fonctionnement natif d'[ember pour l'update d'un model](https://guides.emberjs.com/v3.18.0/models/creating-updating-and-deleting-records/#toc_persisting-records).